docs/en/sql-reference/data-types/int-uint.md

```diff
@@ -1,44 +1,59 @@
 ---
 slug: /en/sql-reference/data-types/int-uint
 sidebar_position: 2
-sidebar_label: UInt8, UInt16, UInt32, UInt64, UInt128, UInt256, Int8, Int16, Int32, Int64, Int128, Int256
+sidebar_label: Int | UInt
+title: Int | UInt Types
 ---
 
-# UInt8, UInt16, UInt32, UInt64, UInt128, UInt256, Int8, Int16, Int32, Int64, Int128, Int256
-
-Fixed-length integers, with or without a sign.
+ClickHouse offers a number of fixed-length integers, with a sign (`Int_`) or without a sign (unsigned `UInt_`) ranging from 8 bytes to 256 bytes.
 
 When creating tables, numeric parameters for integer numbers can be set (e.g. `TINYINT(8)`, `SMALLINT(16)`, `INT(32)`, `BIGINT(64)`), but ClickHouse ignores them.
 
 ## Int Ranges
 
-- `Int8` — \[-128 : 127\]
-- `Int16` — \[-32768 : 32767\]
-- `Int32` — \[-2147483648 : 2147483647\]
-- `Int64` — \[-9223372036854775808 : 9223372036854775807\]
-- `Int128` — \[-170141183460469231731687303715884105728 : 170141183460469231731687303715884105727\]
-- `Int256` — \[-57896044618658097711785492504343953926634992332820282019728792003956564819968 : 57896044618658097711785492504343953926634992332820282019728792003956564819967\]
+Integer types have the following ranges:
```

Comment:
Could we have one section "Integer Ranges" and one section "Integer Aliases" instead of separate sections for signed/unsigned types?